maintenance/annotation_maint.py

```diff
@@ -126,9 +128,14 @@ def download_process_annotations():
                         spc_name_path, an.file_pattern.format(id=chrom_id)
                     )
                     np.savetxt(out_file, intervals, fmt="%d")
-                tf = spc_name_path + f"/{an.id}.tar.gz"
+                tf = spc_name_path + f"/{an.id}_vX.tar.gz"
                 make_tarfile(tf, spc_name_path, "")
                 logger.info("made tarball at " + spc_name_path)
+                logger.info(
+                    "ALERT: need to rename the files (replace 'vX' with "
+                    "appropriate version number to not clobber existing files "
+                    "before upload."
+                )
```

Comment:
```suggestion
                logger.warning(
                    "ALERT: need to rename the files (replace 'vX' with "
                    "appropriate version number to not clobber existing files "
                    "before upload."
                )
```

Reply:
Ran locally, found it easy to ignore. Worth making a warning or some other more notable flag? Not sure what best practices dictates here.

Reply:
Well, the filename itself has `_vX` in it, so they should notice that? Hm, maybe instead it should be `_vCHANGEME`?

Reply:
done